src/Hit/Issue.hs

```diff
@@ -51,13 +53,19 @@ getIssue num = fetchIssue num >>= \case
     Left err -> errorMessage $ show err
     Right is -> putTextLn $ showIssueFull is
 
-showIssueName :: Text -> Issue -> Text
-showIssueName colorCode Issue{..} =
-    arrow <> colorCode <> " [#" <> show @Text issueNumber <> "] " <> resetCode <> issueTitle
+showIssueName :: Text -> Maybe Int -> Issue -> Text
```

Comment:
I propose to implement this code in a bit different design. Instead of passing `Maybe Int` which represents `Maybe` padding, let's pass simply `Int` that is the length of the padding. So in case of `Nothing` it will be just `0`. This will simplify the code. Having `Maybe` doesn't bring any wins here because `Nothing` case doesn't have special handling. It's equivalent to zero padding, so let's just use simpler approach.